packages/mongodb-memory-server-core/src/MongoMemoryServer.ts

```diff
@@ -162,29 +195,46 @@ export default class MongoMemoryServer {
     return true;
   }
 
+  /**
+   * Get Information about the currently running instance, if it is not running it returns "false"
+   */
   getInstanceInfo(): MongoInstanceDataT | false {
-    return this.instanceInfoSync || false;
+    return this.instanceInfoSync ?? false;
   }
 
+  /**
+   * Ensure that the instance is running
+   * -> throws if instance cannot be started
+   */
   async ensureInstance(): Promise<MongoInstanceDataT> {
```

Comment:
at this function i wasnt really sure what should be done, so i changed it to be more readable & understand-able